index.js

```diff
@@ -101,37 +102,42 @@ module.exports = {
 
       // TODO
```

Comment:
Just as a sanity check, you can add a check for `feedIds.length === 0` and call `cb(new Error` in case it's true. Also, you can be more strict about it and check if `feedIds` is an array AND with positive length AND with length less than `$MAX_PERMITTED_ACCORDING_TO_SPEC` AND that every item in the array is an ssb-ref or ssb URI.

Reply:
added some simple sanity checking